import{_ as i,W as o,X as s,Z as e,$ as a,a0 as t,a2 as r,C as c}from"./framework-a4c81d3d.js";const l={},d=e("h1",{id:"data-processing-and-quality-control",tabindex:"-1"},[e("a",{class:"header-anchor",href:"#data-processing-and-quality-control","aria-hidden":"true"},"#"),a(" Data processing and quality control")],-1),h=e("h2",{id:"data-processing",tabindex:"-1"},[e("a",{class:"header-anchor",href:"#data-processing","aria-hidden":"true"},"#"),a(" Data processing")],-1),u={href:"https://qiime2.org/",target:"_blank",rel:"noopener noreferrer"},p=e("code",null,"q2-dada2",-1),f=e("code",null,"ASVs",-1),m=e("code",null,"OTUs",-1),g={href:"https://en.wikipedia.org/wiki/Amplicon_sequence_variant",target:"_blank",rel:"noopener noreferrer"},_=r('<h2 id="quality-control" tabindex="-1"><a class="header-anchor" href="#quality-control" aria-hidden="true">#</a> Quality control</h2><p>Two rounds of quality control procedures were applied after data processing and taxonomy classification analysis.</p><ul><li><p>First, samples (runs) contain only a single taxon (i.e., a species or genus accounts for more than 99.99% of total abundance) will be removed from subsequent analysis. And the abundance information will not be included in <b><span style="color:darkblue;">Mycobiome</span><span style="color:forestgreen;">DB</span></b>.</p></li><li><p>Then, samples (runs) with less than 20,000 reads or less than 50% of the original reads after processing will be marked as QC status = 0, which indicates that the sample is of low quality and should be used with caution.</p></li><li><p>Other samples (runs) will be marked as QC status = 1, which indicates that the sample is of high quality.</p></li></ul><h2 id="taxonomic-assignment-to-the-sequenced-reads" tabindex="-1"><a class="header-anchor" href="#taxonomic-assignment-to-the-sequenced-reads" aria-hidden="true">#</a> Taxonomic assignment to the sequenced reads</h2>',4),b={href:"https://unite.ut.ee/",target:"_blank",rel:"noopener noreferrer"},w=e("h2",{id:"identification-of-genera-co-occurrence",tabindex:"-1"},[e("a",{class:"header-anchor",href:"#identification-of-genera-co-occurrence","aria-hidden":"true"},"#"),a(" Identification of genera co-occurrence")],-1),y=e("p",null,[a("Co-occurred genera may be functionally relavent. In "),e("b",null,[e("span",{style:{color:"darkblue"}},"Mycobiome"),e("span",{style:{color:"forestgreen"}},"DB")]),a(", co-occurred genera pairs were identified for health and each disease separately.")],-1),q={href:"https://github.com/dlegor/SparCC",target:"_blank",rel:"noopener noreferrer"};function x(k,v){const n=c("ExternalLinkIcon");return o(),s("div",null,[d,h,e("p",null,[a("The original ITS data will be processed using "),e("a",u,[a("Qiime2"),t(n)]),a(". "),p,a(" plugin will be applied for Dada2 denoising, resulting in the generation of "),f,a(" (amplicon sequence variants). In this step. the paired-end sequencing reads will be merged automatically. ASVs can be considered as more precise and reproducible than "),m,a(" (operational taxonomic units). ASVs will be used for downstream analysis as an alternative to traditional OTUs. "),e("a",g,[a("Wikipedia: ASV"),t(n)]),a(". Since Dada2 includes internal chimera checking and abundance filtering, no additional filtering steps are required.")]),_,e("p",null,[a("In the process of species classification annotation, we first imported a Qiime2-compatible version of the "),e("a",b,[a("UNITE database"),t(n)]),a(" for training the feature classifier (using Qiime2 plugins). Subsequently, we used the trained classifier to process the obtained ASVs, allowing us to determine the genera abundance composition for each sample. Relative abundances were then calculated for each sample, with the totaling abundance values of 100% respectively.")]),w,y,e("p",null,[a("To investigate the co-occurrence patterns of fungal communities within each disease (or health), we utilized a dataset containing over 150 samples. We employed "),e("a",q,[a("SparCC"),t(n)]),a(" to calculate the co-occurrence between every pair of microbes. Genera with p-values below the threshold of 0.05 were filtered out.")])])}const T=i(l,[["render",x],["__file","dataprocessing.html.vue"]]);export{T as default};
