import{_ as i,W as l,X as d,Z as e,$ as a,a0 as t,a2 as s,C as r}from"./framework-a4c81d3d.js";const c={},h=e("h1",{id:"projects-and-runs",tabindex:"-1"},[e("a",{class:"header-anchor",href:"#projects-and-runs","aria-hidden":"true"},"#"),a(" Projects and runs")],-1),p=e("code",null,"projects and runs",-1),u=e("strong",null,":fontawesome-solid-database: Data",-1),_={href:"https://mycobiome.humangut.info/data",target:"_blank",rel:"noopener noreferrer"},b={href:"https://mycobiome.humangut.info/data/curatedprojects",target:"_blank",rel:"noopener noreferrer"},f=e("code",null,"projects and runs",-1),m={id:"all-projects-and-runs",tabindex:"-1"},g=e("a",{class:"header-anchor",href:"#all-projects-and-runs","aria-hidden":"true"},"#",-1),w={href:"https://mycobiome.humangut.info/data",target:"_blank",rel:"noopener noreferrer"},y=s('<p>This page provides statistics on the data collected in <b><span style="color:darkblue;">Mycobiome</span><span style="color:forestgreen;">DB</span></b>, and lists all <strong>projects</strong> and <strong>runs</strong> in two tables, from the tables users can access webpages for details of a specific project or run.</p><p>It thus consists three parts:</p><h3 id="_1-overview" tabindex="-1"><a class="header-anchor" href="#_1-overview" aria-hidden="true">#</a> 1. Overview</h3><p>This parts provides overview of data collected in our database. For example, the current release contains:</p><ul><li>Meta data for in total 71,642 runs (samples), belonging to 353 projects. Among which, raw data have been processed for in total 53,700 runs (samples), belonging to 282 projects.</li><li>Among which, microbe abundance data are available for in total 37,550 runs (samples), belonging to 209 projects.</li><li>In total 16,150 runs (samples), belonging to 159 projects failed our QC processes.</li><li>In addition, <b><span style="color:darkblue;">Mycobiome</span><span style="color:forestgreen;">DB</span></b> also includes information on 76 projects whose raw data were not processed, including in total 31,381 runs/samples, due to various reasons, mostly the lack of a clearly defined health/disease information.</li></ul><h3 id="_2-all-projects-table" tabindex="-1"><a class="header-anchor" href="#_2-all-projects-table" aria-hidden="true">#</a> 2. All projects table</h3><p>This table contains a list of collected projects, their associated diseases, related publications, brief descriptions and whether the rawdata have been processed.</p><p>The rawdata of a project will not be processed if it lacks certain essential meta-data. For example, <b><span style="color:darkblue;">Mycobiome</span><span style="color:forestgreen;">DB</span></b> currently contains 76 projects whose raw data were not processed, correspondin to in total 31,381 runs/samples, due to various reasons, mostly because of the lack of phenotype information; uses can click the project id for details.</p>',8),j={href:"https://mycobiome.humangut.info/data/project/PRJNA682076",target:"_blank",rel:"noopener noreferrer"},v=e("code",null,"666",-1),k=e("code",null,"disease/heath information, age, gender and BMI",-1),x=s('<h3 id="_3-all-runs-table" tabindex="-1"><a class="header-anchor" href="#_3-all-runs-table" aria-hidden="true">#</a> 3. All runs table</h3><p>This table contains a list of collected samples in <b><span style="color:darkblue;">Mycobiome</span><span style="color:forestgreen;">DB</span></b>. Also available are meta-data including:</p><ul><li><p>technical meta-data such as:</p><ul><li>sequencing devices / instruments, and</li><li>number of obtained sequencing reads.</li></ul></li><li><p>Host-related, biological-relevant meta-data such as:</p><ul><li>disease or health of the host (refered as to <code>disease</code> in our database),</li><li>age,</li><li>sex,</li><li>BMI (body mass index), and</li><li>antibiotic usage.</li></ul></li></ul><p>Meta data are available for in total <code>71,642</code> runs/samples.</p><p>A QCStatus (quanlity-control result) is available for each run, which can be one of the three values:</p><ul><li>1: the data passed our QC procedure and the processed results (i.e., taxnomic assignments) have been loaded into our database,</li><li>0: the data did NOT pass our QC,</li><li>(empty): data yet to be proccessed and loaded to our database.</li></ul>',6),R={href:"/materialsandmethods/dataprocessingandqc/#quality-control",target:"_blank",rel:"noopener noreferrer"},B={id:"project-details-page",tabindex:"-1"},I=e("a",{class:"header-anchor",href:"#project-details-page","aria-hidden":"true"},"#",-1),C=e("p",null,[a("The project details page is available for each collected project in "),e("b",null,[e("span",{style:{color:"darkblue"}},"Mycobiome"),e("span",{style:{color:"forestgreen"}},"DB")]),a(", and provides details on a project, including:")],-1),T=e("ul",null,[e("li",null,"included samples,"),e("li",null,"associated disease(s),"),e("li",null,"related publication(s), and"),e("li",null,"disease marker analysis results")],-1),A=e("p",null,"Here are some random examples:",-1),D={href:"https://mycobiome.humangut.info/data/project/PRJEB1220",target:"_blank",rel:"noopener noreferrer"},P={href:"https://mycobiome.humangut.info/data/project/PRJEB6337",target:"_blank",rel:"noopener noreferrer"},E={href:"https://mycobiome.humangut.info/data/project/PRJEB6070",target:"_blank",rel:"noopener noreferrer"},N=e("h3",{id:"_1-project-overview",tabindex:"-1"},[e("a",{class:"header-anchor",href:"#_1-project-overview","aria-hidden":"true"},"#"),a(" 1. Project overview")],-1),S=e("p",null,"This part provides some basic information regarding a project, including:",-1),M={href:"https://www.ebi.ac.uk/ena/browse",target:"_blank",rel:"noopener noreferrer"},J={href:"https://www.ncbi.nlm.nih.gov/sra",target:"_blank",rel:"noopener noreferrer"},z=e("li",null,"number of included runs,",-1),q=e("li",null,"related publication(s), if available,",-1),L={href:"https://mycobiome.humangut.info/data/curatedprojects",target:"_blank",rel:"noopener noreferrer"},Q=e("h3",{id:"_2-included-runs-samples",tabindex:"-1"},[e("a",{class:"header-anchor",href:"#_2-included-runs-samples","aria-hidden":"true"},"#"),a(" 2. Included runs/samples")],-1),H=e("p",null,[a("This part includes a table that contains a list of runs/samples included in the project. Similar to the "),e("code",null,"All runs table"),a(", related technical and biological meta-data and QCStatus are also listed in the table. Please consult the "),e("code",null,"All runs table"),a(" section above for more details.")],-1),V=e("h3",{id:"_3-in-depth-analysis",tabindex:"-1"},[e("a",{class:"header-anchor",href:"#_3-in-depth-analysis","aria-hidden":"true"},"#"),a(" 3. In-depth analysis")],-1),F=e("code",null,"In-depth analysis",-1),U=e("code",null,"marker identification",-1),O={href:"/materialsandmethods/indepthanalyses",target:"_blank",rel:"noopener noreferrer"},G=e("p",null,"Briefly, microbial markers that show significantly differential abundances between:",-1),W=e("ul",null,[e("li",null,"either a disease and the healthy control group, e.g., CRC and health,"),e("li",null,"or two distinct stages of a disease, e.g., adenoma and CRC")],-1),X=e("code",null,"LEfSe (Linear discriminant analysis effect size)",-1),Z={href:"https://www.ncbi.nlm.nih.gov/pubmed/21702898",target:"_blank",rel:"noopener noreferrer"},$=e("p",null,[a("In "),e("b",null,[e("span",{style:{color:"darkblue"}},"Mycobiome"),e("span",{style:{color:"forestgreen"}},"DB")]),a(", markers are identified on per-project basis, thus the analysis results are included in this "),e("code",null,"project details"),a(" for each project.")],-1),K=e("p",null,"In addition to be shown in a data table, the LEfSe results are also visualized in a barplot, as shown below:",-1),Y=e("figure",null,[e("img",{src:"https://github.com/evolgeniusteam/gmrepodocumentation/tree/gh-pages/websiteandUI/images/projectsandruns/phenotype_combinations_of_species_of_interests.png",alt:"",tabindex:"0",loading:"lazy"}),e("figcaption")],-1),ee=e("code",null,"|LDA score| >= 2.0",-1),ae=e("code",null,"CRC and healthy controls",-1),te={href:"https://mycobiome.humangut.info/data/project/PRJDB4176",target:"_blank",rel:"noopener noreferrer"},oe=s('<div class="hint-container note"><p class="hint-container-title">Note</p><ul><li>for whole-genome shortgun sequencing (a.k.a mNGS) projects, markers were identified at both <code>species</code> and <code>genus</code> levels;</li><li>for 16S amplicon data (a.k.a 16S) projects, markers were identified at only <code>genus</code>.</li><li>users can choose to show either <code>species</code> or <code>genus</code> level markers, or both, by using the <code>control buttons</code> above the barplot.</li></ul></div>',1),ne={id:"run-details-page",tabindex:"-1"},se=e("a",{class:"header-anchor",href:"#run-details-page","aria-hidden":"true"},"#",-1),re=s('<p>This page shows details of a specific run, and includes two parts.</p><h3 id="_1-run-details-and-meta-data" tabindex="-1"><a class="header-anchor" href="#_1-run-details-and-meta-data" aria-hidden="true">#</a> 1. Run details and meta-data</h3><p>This part includes:</p><ul><li>run ID,</li><li>a brief introduction,</li><li>related project ID and sample ID,</li><li>associated disease, and</li><li>meta-data</li></ul><h3 id="_2-taxonomic-profile" tabindex="-1"><a class="header-anchor" href="#_2-taxonomic-profile" aria-hidden="true">#</a> 2. Taxonomic profile</h3><p>This part shows the taxonomic profile, i.e., the bacterial species/genus identified in the run and their relative abundances.</p>',6),ie={href:"https://mycobiome.humangut.info/data/run/ERR481097",target:"_blank",rel:"noopener noreferrer"},le=e("p",null,"The relative abundances are visualized as below:",-1),de=e("figure",null,[e("img",{src:"https://github.com/evolgeniusteam/gmrepodocumentation/tree/gh-pages/websiteandUI/images/projectsandruns/run_taxon_profile.png",alt:"",tabindex:"0",loading:"lazy"}),e("figcaption")],-1),ce=e("p",null,"Users can download the detailed profile as a text file using the link below the figures.",-1);function he(pe,ue){const o=r("ExternalLinkIcon"),n=r("HopeIcon");return l(),d("div",null,[h,e("p",null,[a("The "),p,a(" webpages can be found from the "),u,a(" menu, the latter contains two submenus, one for "),e("a",_,[a(":fontawesome-solid-database: all projects and runs"),t(o)]),a(", the other is for "),e("a",b,[a("curated projects"),t(o)]),a(".")]),e("p",null,[a("There are four types of "),t(n,{icon:"page",color:"gray",size:"24px"}),a(" webpages for "),f,a(".")]),e("h2",m,[g,a(),e("a",w,[t(n,{icon:"page",color:"gray",size:"24px"}),a(" All projects and runs"),t(o)])]),y,e("p",null,[a("For example, project "),e("a",j,[a("PRJNA682076"),t(o)]),a(" contains "),v,a(" samples/runs; it was not processed because host-related information were not available, including the "),k,a(".")]),x,e("p",null,[a("Please consult "),e("a",R,[a("Data processing & QC"),t(o)]),a(" for more details.")]),e("h2",B,[I,a(),t(n,{icon:"page",color:"gray",size:"24px"}),a(" project details page")]),C,T,A,e("ul",null,[e("li",null,[e("a",D,[a("PRJEB1220"),t(o)])]),e("li",null,[e("a",P,[a("PRJEB6337"),t(o)])]),e("li",null,[e("a",E,[a("PRJEB6070"),t(o)])])]),N,S,e("ul",null,[e("li",null,[a("a brief project description obtained from public databases, mostly from "),e("a",M,[a("ENA (European Nucleotide Archive)"),t(o)]),a(" or "),e("a",J,[a("NCBI SRA (Sequence Read Archive) database"),t(o)]),a(",")]),z,q,e("li",null,[a("whether the project is a "),e("a",L,[a("curated project"),t(o)]),a(".")])]),Q,H,V,e("p",null,[F,a(" so far includes only "),U,a(". Please consult our documentation on "),e("a",O,[a("Disease marker identification"),t(o)]),a(" for more details.")]),G,W,e("p",null,[a("are identified using "),X,e("a",Z,[a("PMID: 21702898 "),t(o)]),a(".")]),$,K,Y,e("p",null,[a("this plot shows marker species that show "),ee,a(" between "),ae,a(" in project "),e("a",te,[a("PRJDB4176"),t(o)]),a(", in which green (pink) bars indicate health(CRC)-enriched species.")]),oe,e("h2",ne,[se,a(),t(n,{icon:"page",color:"gray",size:"24px"}),a(" Run details page")]),re,e("p",null,[a("See "),e("a",ie,[a("ERR481097"),t(o)]),a(" for an example.")]),le,de,ce])}const be=i(c,[["render",he],["__file","projectandrunpages.html.vue"]]);export{be as default};
